integration-tests/tests/run-notebook.js

```diff
@@ -9,50 +9,80 @@ const testRunNotebookFn = _.flow(
   withWorkspace,
   withBilling,
   withRegisteredUser
-)(async ({ workspaceName, page, testUrl, token }) => {
+)(async ({ billingProject, workspaceName, page, testUrl, token }) => {
+  await testRunNotebookHelper(billingProject, workspaceName, page, testUrl, token)
+})
+
+const testRunNotebookWithV1WorkspaceFn = _.flow(
+  withV1Workspace,
+  withBilling,
+  withRegisteredUser
+)( async ({ billingProject, v1WorkspaceName, page, testUrl, token }) => {
+  await testRunNotebookHelper(billingProject, v1WorkspaceName, page, testUrl, token)
+})
+
+const testRunNotebookHelper = async (billingProject, workspaceName, page, testUrl, token) => {
   await page.goto(testUrl)
   await click(page, clickable({ textContains: 'View Workspaces' }))
   await signIntoTerra(page, token)
   await dismissNotifications(page)
   await findElement(page, clickable({ textContains: workspaceName }))
   await waitForNoSpinners(page)
   await click(page, clickable({ textContains: workspaceName }))
-  await delay(5000)
+  await delay(10000)
```

Comment:
This was bumped up due to an issue with timing on asynchronous permissions updates (namely google granting bucket access)